lib/ClangImporter/ClangImporter.cpp

```diff
@@ -68,7 +68,9 @@ using namespace importer;
 using clang::CompilerInstance;
 using clang::CompilerInvocation;
 
+#if !defined(_MSC_VER)
 #pragma mark Internal data structures
+#endif
```

Comment:
Not every warning exists for a practical reason for all projects. `#pragma mark` is a pretty common de-facto standard for IDE navigation, and it would be impractical to guard every use of it.